homeassistant/components/light/mqtt.py

```diff
@@ -563,6 +569,27 @@ def supported_features(self):
             if self._optimistic_brightness:
                 self._brightness = kwargs[ATTR_BRIGHTNESS]
                 should_update = True
+        elif ATTR_BRIGHTNESS in kwargs and \
```

Comment:
We should include `ATTR_HS_COLOR not in kwargs` here, since if both color and brightness are specified in a request, the RGB topic will have already been published with this brightness taken into account, so we don't want to publish twice.